src/super_gradients/training/utils/quantization_utils.py

```diff
@@ -261,31 +262,21 @@ def _validate_args(self, start_epoch: int, quant_modules_calib_method: str, cali
 
     def __call__(self, context: PhaseContext):
         if context.epoch == self.start_epoch:
-            # SET CHECKPOINT PARAMS SO WE LOAD THE BEST CHECKPOINT SO FAR
```

Comment:
why was that removed?

Reply:
Checkpoint_params aren't used anymore since build_model is removed (when using get, with a SgTrainer checkpoint it will automatically take ema net. Other checkpoint params aren't relevant here either..bottom line its the same behaviour)